esp/src/eclwatch/DFUQueryWidget.js

```diff
@@ -345,6 +345,30 @@ define([
 
             //  Implementation  ---
             getFilter: function () {
+                var context = this;
+                if (context.workunitsGrid){
+                var retVal = this.filter.toObject();
```

Comment:
Indent everything inside the if. Also, why is this `if` needed given that getFilter() is indeed called by the init of that workunitsGrid.

Reply:
it is needed because before the page is loaded, workunitsGrid does not exist, which causes this filter option to break.